docker/CONFIGURATION.md

```diff
@@ -0,0 +1,370 @@
+| `port` | string | `"8080"` | Port to listen on |
+| `address` | string | `"127.0.0.1"` | Address to bind to |
+| `baseURL` | string | `""` | Base URL for reverse proxy |
+| `log` | string | `"stdout"` | Log output (`stdout`, `stderr`, or file path) |
+| `database` | string | `"./filebrowser.db"` | Database file path |
+| `root` | string | `"."` | Root directory to serve |
+| `tokenExpirationTime` | string | `"2h"` | Session token expiration time |
+| `socket` | string | `""` | Unix socket path (alternative to port) |
+| `tlsKey` | string | `""` | TLS private key file |
+| `tlsCert` | string | `""` | TLS certificate file |
+
+## Environment Variables
+
+All configuration options can be set via environment variables using the `FB_` prefix:
+
+| Config Key | Environment Variable | Example |
+|------------|---------------------|---------|
+| `token-expiration-time` | `FB_TOKEN_EXPIRATION_TIME` | `FB_TOKEN_EXPIRATION_TIME=6h` |
+| `port` | `FB_PORT` | `FB_PORT=8080` |
+| `address` | `FB_ADDRESS` | `FB_ADDRESS=0.0.0.0` |
+| `baseURL` | `FB_BASEURL` | `FB_BASEURL=/files` |
+| `log` | `FB_LOG` | `FB_LOG=stdout` |
+| `database` | `FB_DATABASE` | `FB_DATABASE=/database/filebrowser.db` |
+| `root` | `FB_ROOT` | `FB_ROOT=/srv` |
+
+**Note:** Both dashes (`-`) and dots (`.`) in config keys are replaced with underscores (`_`) in environment variable names.
+
+## CLI Flags
+
+All server settings can be passed as command-line flags:
+
+```bash
+filebrowser --token-expiration-time 6h \
+            --port 8080 \
+            --address 0.0.0.0 \
+            --root /srv \
+            --database /database/filebrowser.db
+```
+
+## Database Configuration
+
+### Initial Setup
+
+When the container starts for the first time without an existing database:
+
+1. The container copies `/defaults/settings.json` to `/config/settings.json`
+2. File Browser reads the configuration file
+3. A new database is created with these settings
+4. An admin user is created
+
+**Important:** Make sure your `/config/settings.json` has `tokenExpirationTime` set BEFORE the first startup, or it will default to `2h` in the database.
+
+### Updating Existing Database
+
+If you already have a database and want to change the token expiration time:
+
+#### Option A: Use CLI Command (Recommended)
+```bash
+docker exec -it <container_name> filebrowser config set --token-expiration-time 6h
+```
+
+#### Option B: Recreate Database
+1. Stop the container
+2. Delete the database file (`/database/filebrowser.db`)
+3. Update `/config/settings.json` with desired `tokenExpirationTime`
+4. Restart the container (new database will be created)
+
+**Warning:** Option B will delete all users, settings, and shares!
+
+#### Option C: Update Config File and Restart
+1. Update `/config/settings.json` with `tokenExpirationTime`
+2. Restart the container
+
+The configuration file will override the database value on startup (as of the latest version).
+
+## Troubleshooting
```

Comment:
This is nice. We could create a troubleshooting page in the docs and include this things, since they can happen with any configuration